recipe/meta.yaml

```diff
@@ -113,6 +113,7 @@ extra:
     - atmyers
     - ax3l
     - dpgrote
+    - EZoni
```

Comment:
Ping @EZoni: FYI, after merge of this you get a conda-forge invite and become co-maintainer of this repo

Reply:
Thanks, Axel! :pray: